packages/frontend/src/index.scss

```diff
@@ -1,47 +1,68 @@
-/*! Clug Bootstrap 4 theme */
+// Import Bootstrap and its default variables
+$theme-colors: (
+  "primary": #3cddb4,
+  "secondary": #ff581a,
+  "danger": #ff4136
+);
+@import '~bootstrap/scss/bootstrap.scss';
+@import './theming/alerts.scss';
+@import './theming/button.scss';
+
+/*! Clug theme */
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500&display=swap');
 $font-family-base: 'Roboto', sans-serif;
 $headings-font-family: 'Roboto', sans-serif;
-
 $enable-grid-classes: false;
+/* colors */
+
 $primary: #3cddb4;
+$primary-dark: darken($primary, 10%);
+$primary-light: lighten($primary, 10%);
 $secondary: #ff581a;
-$success: #3cddb4;
-$danger: #ff581a;
+$secondary-dark: darken($secondary, 10%);
+$secondary-light: lighten($secondary, 10%);
+$success: #3cdd6a;
+$danger: #c10903;
+$danger-bg: lighten($danger, 55%);
 $info: #0f81c7;
 $warning: #ff581a;
-$light: #ececec;
-$dark: #222222;
+$light-text: #ececec;
+$dark-text: #222222;
+$bg-light: #f5faf9;
+$primary-o-25: rgba(60, 221, 180, 0.25);
+$secondary-o-25:rgba(255, 88, 26, 0.25);
 
-// Import Bootstrap and its default variables
-@import '~bootstrap/scss/bootstrap.scss';
 
 html {
-  /*
-   * CSS variables must be defined with a special scss synthax:
-   * https://sass-lang.com/documentation/breaking-changes/css-vars
-  */
-
-  --clug-primary: #{$primary};
-  --clug-background-color: #f5faf9;
-  --clug-secondary: #{$secondary};
-  --clug-title: 2rem;
-  --clug-subtitle: 1.1rem;
-  --clug-text: 0.8rem;
-  --clug-btn-radius: 20px;
-  margin: 0;
-}
 
-body {
-  margin: 0;
-}
+/* colors */
+  --c-primary: #{$primary};
+  --c-dark-primary: #{$primary-dark};
+  --c-light-primary: #{$primary-light};
+  --c-secondary: #{$secondary};
+  --c-dark-secondary: #{$secondary-dark};
+  --c-light-primary: #{$primary-light};
+  --c-background-color: #{$bg-light};
+  --c-danger: #{$danger};
+  --c-danger-bg: #{$danger-bg};
 
-header {
-  // min-height: 10vh;
-}
+  /* text */
+
+  --c-light-text: #{$light-text};
+  --c-dark-text: #{$dark-text};
+
+/* font sizes */
+  --title-size: 2rem;
+  --subtitle-size: 1.1rem;
+  --text-size: 0.8rem;
+
+/* shapes */
+  --c-radius: 12px;
 
-main {
 }
 
-footer {
+* {
```

Comment:
I'm not sure it's a good idea to apply to all element this css code

Reply:
Yes it is something that we really often do to avoid bad surprises. Anyway, (your loving) bootstrap components will override this poor * tag whenevr it is needed.